File: sample/netcorehost/DomainController.js
var DomainController = function DomainController(cfg)
{
    let ctrl = {};

    ctrl.CheckDomain = function (domainName, onResponse)
    {
        cfg.Connector.Submit(domainName, "ardvro/component/netcorehost/Domain/CheckDomain", function (box)
        {
            onResponse != null ? onResponse(box == null ? null : box.Data) : null;
        });
    };

    ctrl.CheckDomainNs = function (domainName, onResponse)
    {
        cfg.Connector.Submit(domainName, "ardvro/component/netcorehost/Domain/CheckDomainNs", function (box)
        {
            onResponse != null ? onResponse(box == null ? null : box.Data) : null;
        });
    };

    return ctrl;
};

File: sdk/android/netcorehost/DomainController.java
package ardvro.component.netcorehost;

import ardvro.core.connection.IConnection;
import ardvro.core.connection.IResponseListener;

public class DomainController
{
    private IConnection _connection;
    public DomainController(IConnection connection)
    {
        _connection = connection;
    }

    public void CheckDomain(String domainName, IResponseListener<Boolean> callback)
    {
        _connection.Submit(domainName, "ardvro/component/netcorehost/Domain/CheckDomain", callback);
    }

    public void CheckDomainNs(String domainName, IResponseListener<String> callback)
    {
        _connection.Submit(domainName, "ardvro/component/netcorehost/Domain/CheckDomainNs", callback);
    }


}
